### ISSUE TITLE
Saving multiple remote connections

### ISSUE THREAD

**Issue by ryanfurtner** [thebaselab/codeapp#1129]
I just got an iPad Air 5 to be able to do remote work on client websites via FTP when im away from my desk. Code App seems perfect but I assumed I would be able to save more than 1 remote connection? 

Am I doing something wrong? After saving one, disconnecting from it and renaming it, I would love to be able to then add more of my client sites. When I attempt this, it keep just replacing the 1 connection.

**Comment by bummoblizard** [comment]
This looks like a bug to me. Do the servers happen to have the same IP address?

**Comment by ryanfurtner** [comment]
You are correct, they do in fact have the same IP address. I thought that might have been the problem!
